src/config.rs

```diff
@@ -199,21 +199,32 @@ pub enum Config {
     /// The primary email address. Also see `SecondaryAddrs`.
     ConfiguredAddr,
 
+    /// List of configured IMAP servers as a JSON array.
+    ConfiguredImapServers,
+
     /// Configured IMAP server hostname.
+    ///
+    /// This is replaced by `configured_mail_servers` for new configurations.
     ConfiguredMailServer,
 
-    /// Configured IMAP server username.
-    ConfiguredMailUser,
-
-    /// Configured IMAP server password.
-    ConfiguredMailPw,
-
     /// Configured IMAP server port.
+    ///
+    /// This is replaced by `configured_mail_servers` for new configurations.
     ConfiguredMailPort,
 
     /// Configured IMAP server security (e.g. TLS, STARTTLS).
+    ///
+    /// This is replaced by `configured_mail_servers` for new configurations.
     ConfiguredMailSecurity,
 
+    /// Configured IMAP server username.
+    ///
+    /// This is replaced by `configured_mail_servers` for new configurations.
+    ConfiguredMailUser,
+
+    /// Configured IMAP server password.
+    ConfiguredMailPw,
```

Comment:
Should be renamed to `ConfiguredImapPw` for consistency with `ConfiguredImapServers`